src/views/tickets/orders-rsvp.php

```diff
@@ -17,57 +17,79 @@
 $post      = get_post( $post_id );
 $post_type = get_post_type_object( $post->post_type );
 $user_id   = get_current_user_id();
+$user_info = get_userdata( $user_id );
 $attendees = $view->get_event_rsvp_attendees( $post_id, $user_id );
 
+$order_id = wp_list_pluck( $attendees, 'order_id'  );
+$product_id = wp_list_pluck( $attendees, 'product_id'  );
+$optout = wp_list_pluck( $attendees, 'optout'  );
+$name = wp_list_pluck( $attendees, 'purchaser_name' );
+$email = wp_list_pluck( $attendees, 'purchaser_email' );
+$time = wp_list_pluck( $attendees, 'purchase_time' );
```

Comment:
Now that I see the changeset in more detail, I'll make the following suggestion:

Rather than doing a bunch of `wp_list_pluck()` calls, just grab the first attendee since that seems to be the only one you actually care about down below.

```php
// move this below the if ( ! $view->has_rsvp_attendees( $post_id, $user_id ) ) { block
$first_record = $attendees[0];
```

Then when referencing the order_id, optout, email, etc, you can do:

```php
$first_record['order_id']
```